<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableWindow</class>
 <widget class="QWidget" name="TableWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>952</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>History viewer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../../resources/nicos-gui.qrc">
    <normaloff>:/find</normaloff>:/find</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="opaqueResize">
      <bool>true</bool>
     </property>
     <property name="handleWidth">
      <number>6</number>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>History views:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="viewList">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="plotFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="plotLayout"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="../../../../resources/nicos-gui.qrc">
     <normaloff>:/print</normaloff>:/print</iconset>
   </property>
   <property name="text">
    <string>&amp;Print...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="../../../../resources/nicos-gui.qrc">
     <normaloff>:/quit</normaloff>:/quit</iconset>
   </property>
   <property name="text">
    <string>&amp;Close window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionUnzoom">
   <property name="icon">
    <iconset resource="../../../../resources/nicos-gui.qrc">
     <normaloff>:/viewfull</normaloff>:/viewfull</iconset>
   </property>
   <property name="text">
    <string>&amp;Unzoom</string>
   </property>
  </action>
  <action name="actionSavePlot">
   <property name="icon">
    <iconset resource="../../../../resources/nicos-gui.qrc">
     <normaloff>:/save</normaloff>:/save</iconset>
   </property>
   <property name="text">
    <string>&amp;Save plot</string>
   </property>
   <property name="toolTip">
    <string>Save plot in portable format</string>
   </property>
  </action>
  <action name="actionLogScale">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Logarithmic Y scale</string>
   </property>
   <property name="iconText">
    <string>Log Scale</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionCloseView">
   <property name="text">
    <string>Close view</string>
   </property>
   <property name="iconText">
    <string>Close</string>
   </property>
   <property name="toolTip">
    <string>Close plot for this view</string>
   </property>
  </action>
  <action name="actionLegend">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>L&amp;egend</string>
   </property>
  </action>
  <action name="actionDeleteView">
   <property name="text">
    <string>Delete view</string>
   </property>
   <property name="iconText">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete view</string>
   </property>
  </action>
  <action name="actionResetView">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../../../../resources/nicos-gui.qrc">
     <normaloff>:/new</normaloff>:/new</iconset>
   </property>
   <property name="text">
    <string>New view</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionAttachElog">
   <property name="text">
    <string>Attach to electronic log</string>
   </property>
  </action>
  <action name="actionSymbols">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show symbols</string>
   </property>
   <property name="iconText">
    <string>Symbols</string>
   </property>
  </action>
  <action name="actionLines">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show lines</string>
   </property>
   <property name="iconText">
    <string>Lines</string>
   </property>
  </action>
  <action name="actionEditView">
   <property name="icon">
    <iconset resource="../../../../resources/nicos-gui.qrc">
     <normaloff>:/editor</normaloff>:/editor</iconset>
   </property>
   <property name="text">
    <string>Edit view</string>
   </property>
  </action>
  <action name="actionLinearFit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Linear fit</string>
   </property>
  </action>
  <action name="actionSaveData">
   <property name="icon">
    <iconset resource="../../../../resources/nicos-gui.qrc">
     <normaloff>:/table</normaloff>:/table</iconset>
   </property>
   <property name="text">
    <string>Save data</string>
   </property>
   <property name="toolTip">
    <string>Save the data in plain text file</string>
   </property>
  </action>
  <action name="actionAutoScale">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto scale</string>
   </property>
   <property name="toolTip">
    <string>autoscale: growing window</string>
   </property>
  </action>
  <action name="actionScaleX">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>X</string>
   </property>
   <property name="toolTip">
    <string>Automatic scaling in x direction</string>
   </property>
  </action>
  <action name="actionScaleY">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
   <property name="toolTip">
    <string>Automatic scaling in y direction</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../../resources/nicos-gui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>TableWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
